src/objective-c/GRPCClient/private/GRPCCompletionQueue.h

```diff
@@ -32,11 +32,11 @@
  */
 
 #import <Foundation/Foundation.h>
+#include <grpc/grpc.h>
 
 struct grpc_completion_queue;
```

Comment:
nit: We don't need to declare this one anymore now that we're including the whole `grpc.h`, right?